### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kristelfung**
hi

### CODE REVIEW

**Comment by kristelfung** [comment]
hi pls accept

File: README.md [hunk 1]
```diff
@@ -29,3 +29,4 @@ To contribute, clone the repo and run
 
 | Username | Year | Favorite food |
 |----------|------|---------------|
+| kristelfung | 2022 | Sushi |
```
Comment (NicholasLYang) [inline README.md]: make this prettier

Comment (kristelfung) [reply README.md]: why

Comment (kristelfung) [reply README.md]: ok

**Comment by themichaelyang** [comment]
very nice

**Comment by kristelfung** [comment]
> very nice

thank you!

**Pull request merged**